### PR TITLE
fix: clean up settings

### PR DESCRIPTION
**Pull request by Sirs0ri**
functional fixes (via the change in src/utils/utils.ts)
* markername / iconname inputs no longer disappear when empty
* markername / iconname inputs now show errors correctly

visual fixes (via the changes in src/modals/settings.ts / main.css):
* remove 2px "double borders" between some settings
* remove unnecessary borders at the top/bottom of "create/edit marker" dialog
* show big preview of images used as markers

### CODE REVIEW

**Comment by sigrunixia** [comment]
Would this close any existing issues, @Sirs0ri?

**Comment by Sirs0ri** [comment]
@sigrunixia nope

I thought about opening one for the disappearing markername/iconname fields in the settings ui, but then realised it would be faster to directly submit a PR with the fix.

if you want me to open an issue first I'm happy to do that!

File: src/assets/main.css [hunk 1]
```diff
@@ -74,6 +75,15 @@
     border-top: none;
 }
 
+.icon-display {
```
Comment (valentine195) [inline src/assets/main.css:78]: While I see the intent of this, in effect its stretching a resized image, making it look blurry.


I will fix this in the canvas itself.

**Comment by sigrunixia** [comment]
> @sigrunixia nope
> 
> I thought about opening one for the disappearing markername/iconname fields in the settings ui, but then realised it would be faster to directly submit a PR with the fix.
> 
> if you want me to open an issue first I'm happy to do that!

Oh no worries! It's nice to have issues beforehand, or close tags in the description, but I was double checking for our project board. :)

**Pull request merged**